fdb-record-layer-core/fdb-record-layer-core.gradle

```diff
@@ -28,37 +28,37 @@ if (!hasProperty('coreNotStrict')) {
 }
 
 configurations {
-    repl.extendsFrom testRuntime
+    repl.extendsFrom testRuntimeOnly
 }
 
 dependencies {
-    compile project(':fdb-extensions')
+    api project(':fdb-extensions')
```

Comment:
I _think_ we would want this to be `implementation` instead of `api`, at least at some point, as I don't think the Record Layer needs to/should expose the extended FDB in its API. I'm also fine with keeping this as `api` for now to avoid accidentally disturbing anyone downstream. There are certainly some utilities in `fdb-extensions` that an adopter might be using, though they should arguably just add an explicit dependency.

Reply:
I am happy with `api` for `fdb-java` and `implementation` for `fdb-extensions`. I think it depends on whether you think the "extended" FDB is an "enhanced" version or orthogonal additions.

Reply:
changed to `implementation` and added `fdb-extensions` (as `implementation`) to projects that needed it.